app/controllers/events/view/speakers/list.js

```diff
@@ -90,8 +90,8 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
   }
 
   @action
-  viewSpeaker(id) {
-    this.transitionToRoute('events.view.speakers.edit', id);
+  viewSpeaker(speaker) {
+    this.transitionToRoute('public.speaker.view', speaker.event.get('id'), speaker.id);
```

Comment:
```suggestion
    this.transitionToRoute('public.speaker.view', speaker.event.get('identifier'), speaker.id);
```

Reply:
Should I make this change? bcz it is working both ways